integration_tests/pages/refer/selectProgramme.ts

```diff
@@ -0,0 +1,35 @@
+import Page from '../page'
+import type { Course } from '@accredited-programmes/models'
+
+export default class SelectProgrammePage extends Page {
+  courses: Array<Course>
+
+  constructor(args: { courses: Array<Course> }) {
+    // Conditional radio buttons add an additional `aria-expanded` field,
+    // so ignore that rule on this page
+    super('Add Accredited Programme history', { accessibilityRules: { 'aria-allowed-attr': { enabled: false } } })
```

Comment:
Is it worth making the exception more specific, e.g. to a particular selector like here: https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/blob/main/integration_tests/support/commands.ts#L29. We still want to check this rule outside the conditional input, right?

Reply:
Yeah possibly, might need to make a change to `Cypress.Commands.add('checkAccessibility`.  Will investigate separately.